quantlib/models/equity/heston_model.pyx

```diff
@@ -29,21 +30,23 @@ from quantlib.time.date cimport Period
 from quantlib.termstructures.yields.flat_forward cimport (
     YieldTermStructure
 )
+from quantlib.models.calibration_helper cimport CalibrationHelper
+
 
 cdef public enum CALIBRATION_ERROR_TYPE:
-    RelativePriceError = _hm.RelativePriceError
-    PriceError = _hm.PriceError
-    ImpliedVolError = _hm.ImpliedVolError
+    RelativePriceError = _ch.RelativePriceError
+    PriceError = _ch.PriceError
+    ImpliedVolError = _ch.ImpliedVolError
 
-cdef class HestonModelHelper:
+cdef class HestonModelHelper(CalibrationHelper):
 
     def __cinit__(self):
-        self._thisptr = NULL
-
-    def __dealloc__(self):
-        if self._thisptr is not NULL:
-            # print('heston dealloc')
-            del self._thisptr
+        pass
+    
+    ## def __dealloc__(self):
```

Comment:
Can you remove the commented code? I guess you commented it so that it uses the one from the `CalibrationHelper`?